src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -108,6 +108,9 @@ public TokenAcquisition(
             _credentialsLoader = credentialsLoader;
             _certificatesObserver = serviceProvider.GetService<ICertificatesObserver>();
             tokenAcquisitionExtensionOptionsMonitor = serviceProvider.GetService<IOptionsMonitor<TokenAcquisitionExtensionOptions>>();
+
+            _miHttpFactory = serviceProvider.GetService<IManagedIdentityHttpClientFactory>()
+                   ?? new DefaultManagedIdentityHttpClientFactory(httpClientFactory);
```

Comment:
blocking: By providing a default MSALHttpClient factory that is just an adapter over IHttpClientFactory from ASP.NET Core, you are taking a hard dependency on the HTTP behavior. We really want to avoid that because: 

- ServiceFabric has extra logic to validate SSL certs
- MSIv2 will introduce mTLS connections

The better path forward is to check for null?